pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -449,18 +469,20 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 			// same way it handles the "istio-ca-secret" secret. Isitod utilizes a secret watch instead
 			// of file watch to check for secret updates. This may change in the future, and istiod
 			// will watch the file mount instead.
+
+			// TODO(costin): we can use the presence of intermediary certs to know if we can rotate.
```

Comment:
We don't really care if Istiod generated the the private key and root cert - what is important is ability to rotate the private key and root cert.  For the istio-generated - we can do that because it is a self-signed cert, i.e. the private key we have can be used to generate another root cert. 

If an intermediary cert is present - it means the root cert is signed by another key we don't have - so there is no way for us to rotate the root cert, we can't sign it. 

I don't know why we use the term 'plugged in' - but it is really a cert signed by an external private key vs a self-signed certificate, and the difference is that in the last case we have access to the private key needed to sign the cert and rotate, while in the first case we don't and there's nothing we can do.

Reply:
'plugged in' indicates we can do nothing about it on rotate. Much clearer than some term else